frontend/src/app/modules/grids/widgets/header/header.component.html

```diff
@@ -1,8 +1,5 @@
 <h3 class="widget-box--header"
     [ngClass]="{ '-editable': isRenameable }">
-  <i class="widget-box--header-icon icon-context"
-     aria-hidden="true"
-     [ngClass]="iconClass"></i>
```

Comment:
Please remove this method, as well as the input `icon` in the component. Further the `widgets/wp-table.component` still uses the `icon` - input, so please remove this one, too.

Reply:
Thanks, solved.